bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -71,6 +74,7 @@
     private String callStatusReportPath;
     private String callTimingReportPath;
 
+    private boolean forceJdkTlsProvider = false;
```

Comment:
prefer explicit configuration values of { JDK, OPENSSL }, then map them to SslProvider.JDK and SslProvider.OPENSSL.